kubectl-rook-ceph.sh

```diff
@@ -27,21 +27,23 @@ function print_usage () {
   echo "USAGE"
   echo "  kubectl rook-ceph <main args> <command> <command args>"
   echo "MAIN ARGS"
-  echo "  -h, --help                  : output help text"
-  echo "  -n, --namespace='rook-ceph' : the namespace of the CephCluster"
+  echo "  -h, --help                                : output help text"
+  echo "  -n, --namespace='rook-ceph'               : the namespace of the CephCluster"
+  echo "  -op-ns, --operator-namespace='rook-ceph'  : the namespace of the CephCluster"
```

Comment:
How about we keep the short names to one (or maybe two) letters?
```suggestion
  echo "  -o, --operator-namespace='rook-ceph'  : the namespace of the rook operator"
```